packages/graphql-hooks/test-jsdom/integration/useQuery.test.tsx

```diff
@@ -77,7 +77,7 @@ describe('useQuery Integrations', () => {
     // 3. explict rerender call
     // 4. loading again
     // 5. data v2
-    expect(testComponentRenderCount).toBe(5)
+    expect(testComponentRenderCount).toBe(6)
```

Comment:
sounds flaky (as the others)

Reply:
I think that these changes on the counts depends on new React's rendering

Reply:
I agree, because of that I think its a flaky test